modules-js/react-fleet/src/form-elements/SelectDropdown.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { SelectHTMLAttributes } from 'react';
+import hash from 'string-hash';
+
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
+  label: string;
+  error?: string | boolean;
+
+  variant?: 'small';
```

Comment:
I’d recommend making `small` its own optional boolean prop. That lets you mix variants more easily.